test-libz-rs-sys/src/lib.rs

```diff
@@ -409,7 +409,7 @@ mod null {
             let mut strm = MaybeUninit::<z_stream>::zeroed();
 
             fn initialize_header_null() -> MaybeUninit<gz_header> {
-                let mut head = MaybeUninit::<gz_header>::uninit();
+                let mut head = MaybeUninit::<gz_header>::zeroed();
 
```

Comment:
this test uses some low-level operations still, but we now assume that the memory is initialized, so we have to use `zeroed` here.